drivers/usb/device/usb_dc_sam.c

```diff
@@ -891,7 +887,7 @@ int usb_dc_ep_read_continue(uint8_t ep)
 		return -ENODEV;
 	}
 
-	if (EP_ADDR2DIR(ep) != USB_EP_DIR_OUT) {
+	if (USB_EP_GET_DIR(ep) != USB_EP_DIR_OUT) {
```

Comment:
s/a

Reply:
No, I will not change this logic here. Also not relevant to this PR.